<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Aoip</class>
 <widget class="QWidget" name="Aoip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" columnstretch="3,10">
   <item row="0" column="1">
    <widget class="QPushButton" name="btnConnect">
     <property name="minimumSize">
      <size>
       <width>2</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>CONNECT</string>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="meters" rowstretch="9,1" columnstretch="1,1">
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Output Studio</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QWidget" name="outputMeter" native="true"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Input Local</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QWidget" name="inputMeter" native="true">
       <property name="minimumSize">
        <size>
         <width>87</width>
         <height>356</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Sortie</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QTextBrowser" name="consoleOutput"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
